it/v2/endpoints/RetrieveBISSControllerISpec.scala

```diff
@@ -51,13 +51,22 @@ class RetrieveBISSControllerISpec extends IntegrationBaseSpec with RetrieveBISSF
         .withHttpHeaders(
           (ACCEPT, "application/vnd.hmrc.2.0+json"),
           (AUTHORIZATION, "Bearer 123") // some bearer token
-      )
+        )
     }
+
+  }
+
+  trait TysTest extends Test {
+    def downstreamUrl: String = s"individuals/self-assessment/income-summary/$nino/$incomeSourceType/23-24/$businessId"
+  }
+
+  trait NonTysTest extends Test {
+    def downstreamUrl: String = s"/income-tax/income-sources/nino/$nino/$incomeSourceType/$downstreamTaxYear/biss"
```

Comment:
Maybe also include a `def queryParams: Seq[(String, String)]` in each of these traits so that the trait completely controls the stubbing (whether or not it's TYS). (As it is you need to choose a trait _and_ the query params separately for it to work.)